docs/sources/plugins/developing/dataframe.md

```diff
@@ -10,9 +10,9 @@ draft = true
 
 ## Introduction to data frames
 
-Grafana supports a variety of different data sources, each with its own data model. To make this possible, Grafana consolidates the query results from each of these data sources into one, unified data structure called _data frames_.
+Grafana supports a variety of different data sources, each with its own data model. To make this possible, Grafana consolidates the query results from each of these data sources into one, unified data structure called a _data frame_.
 
-Data frames were introduced in Grafana 7.0 to replace the Time series and Table structures with a more generic data structure that can support a wider range of data types.
+Data frames were introduced in Grafana 7.0 to replace the Time series and Table structures with a more generic data structure that can support a wider range of data types. The data frame structure is a concept that's borrowed from data analysis tools like the [R programming language](https://www.r-project.org), and [Pandas](https://pandas.pydata.org/).
```

Comment:
I recommend removing the "Grafana 7.0" text here. It ages FAST in prose. Instead, put "Note: Only applies to Grafana 7.0+. Previous versions used tables." or something similar near the top. Try to match what is in other topics with features that are version-dependent.

Do _not_ put this as the first paragraph, where it might appear as the text of search results.